h5p-bildetema/src/components/LanguageDropdown/LanguageDropdown.tsx

```diff
@@ -45,7 +47,7 @@ export const LanguageDropdown: React.FC<LanguageDropdownProps> = ({
   };
 
   React.useEffect(() => {
-    setIsActive(false);
+    setIsActive(firstTime);
```

Comment:
This was used to hide the dropdown when the language changes (when a language is selected in the dropdown) I see that the dropdown is now still visible after the language changes.

Reply:
Gotcha, it should be fixed now and works as attended. I will review and refactor my code tomorrow.

Reply:
Works now! ✨